pom.xml

```diff
@@ -47,8 +47,8 @@
     <scm>
         <connection>scm:git:git://github.com/jenkinsci/envinject-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/envinject-plugin.git</developerConnection>
-      <tag>HEAD</tag>
-  </scm>
+        <tag>HEAD</tag>
```

Comment:
It's up to you (not important). 

BTW I suspect this change may not help, because this tag is being automatically injected by mvn release.